### PR TITLE
New package: ferium-4.4.0

### PR DESCRIPTION
**Pull request by yoshiyoshyosh**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, `x86_64_glibc`
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - armv7l (crossbuild)

### CODE REVIEW

**Comment by tranzystorek-io** [comment]
Shell completions are available via `ferium complete <shell>`

File: srcpkgs/ferium/template [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+pkgname=ferium
+version=4.3.4
+revision=1
+build_style=cargo
+short_desc="CLI program for managing Minecraft mods and modpacks"
+maintainer="yosh <yosh-git@riseup.net>"
+license="MPL-2.0"
+homepage="https://github.com/gorilla-devs/ferium"
+changelog="https://raw.githubusercontent.com/gorilla-devs/ferium/main/CHANGELOG.md"
+distfiles="https://github.com/gorilla-devs/ferium/archive/v${version}.tar.gz"
```
Comment (tranzystorek-io) [inline srcpkgs/ferium/template:11]:
```suggestion
distfiles="https://github.com/gorilla-devs/ferium/archive/refs/tags/v${version}.tar.gz"
```

File: srcpkgs/ferium/template [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+	vtargetrun ${FERIUM} complete bash > ferium.bash
+	vtargetrun ${FERIUM} complete zsh > ferium.zsh
+	vtargetrun ${FERIUM} complete fish > ferium.fish
+}
+
+post_install() {
+	vcompletion ferium.bash bash
+	vcompletion ferium.zsh zsh
+	vcompletion ferium.fish fish
+}
```
Comment (tranzystorek-io) [inline srcpkgs/ferium/template:26]: See e.g. https://github.com/void-linux/void-packages/blob/6795b29716be848becdfc53f176f85820c29f7de/srcpkgs/fd/template for an example how it's preferred to be done

Comment (yoshiyoshyosh) [reply srcpkgs/ferium/template:26]: ah, I guess I looked at some un-updated templates x)

**Pull request merged**